src/mito2/src/schedule/scheduler.rs

```diff
@@ -61,34 +103,36 @@ impl LocalScheduler {
     ///
     /// concurrency: the number of bounded receiver
     pub fn new(concurrency: usize) -> Self {
-        let (tx, rx) = async_channel::unbounded();
+        let (tx, rx) = priority::unbounded::<Job>();
```

Comment:
Maybe we need a gauge for the number of pending jobs.

Reply:
> Maybe we need a gauge for the number of pending jobs.

fixed in f3fc31708a